app/javascript/components/GuitarInfo.js

```diff
@@ -20,18 +79,88 @@ const GuitarInfo = ({ name, url, price, description }) => {
           </GridItem>
           <GridItem>
             <CardTitle>{name}</CardTitle>
-            <CardBody>Price: {price}</CardBody>
-            <CardFooter>Description: {description}</CardFooter>
+            <CardBody>
+              Price: {price} <br />
+              Description: {description}
+            </CardBody>
+            <CardFooter>
+              <Button variant="secondary" onClick={() => editGuitar()}>
+                Edit
+              </Button>
+            </CardFooter>
           </GridItem>
         </Grid>
       </Card>
+
+      <Modal
+        variant={ModalVariant.small}
+        title="Edit Guitar"
+        description="Change the fields you wish to change and then click 'Update'"
+        isOpen={isFormModalOpen}
+        onClose={toggleFormModal}
+        actions={[
+          <Button
+            key="update"
+            variant="primary"
+            form="modal-with-form-form"
+            onClick={() => updateGuitar(id)}>
+            Update
+          </Button>,
+          <Button key="cancel" variant="link" onClick={toggleFormModal}>
+            Cancel
+          </Button>
+        ]}>
+        <Form id="modal-with-form-form">
+          <FormGroup label="Name" isRequired fieldId="modal-with-form-form-name">
+            <TextInput
+              isRequired
+              type="string"
+              id="modal-with-form-form-name"
+              name="modal-with-form-form-name"
+              value={nameValue}
+              onChange={(value) => setNameValue(value)}
```

Comment:
nit: when you're using only the first function param, you can write a shortcut:
`onChange={setNameValue}`

Reply:
Oh cool
I haven't known that 
I'll use it 👍🏼